diagnostics/collector.go

```diff
@@ -165,16 +184,53 @@ func (c *DBCollector) writeRawQueryContent(ctx context.Context, writer *zip.Writ
 	return err
 }
 
-func (c *DBCollector) writeTableContent(ctx context.Context, writer *zip.Writer, name string) error {
-	q := sq.Select("*").From(name)
-	return c.writeQueryContent(ctx, writer, name, q)
-}
-
-func (c *DBCollector) writeTableCount(ctx context.Context, writer *zip.Writer, name string) error {
+func (c *Collector) writeTableCount(ctx context.Context, writer *zip.Writer, name string) error {
 	q := sq.Select("COUNT(*)").From(name)
 	return c.writeQueryContent(ctx, writer, name+"_count", q)
 }
 
+func (c *Collector) rangesStats(ctx context.Context, writer *zip.Writer) []error {
```

Comment:
same as before I think the multierror will be a better solution than the slice of errors

Reply:
Done.